src/niscope/examples/niscope_fetch_forever.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+
+import argparse
+import datetime
+import niscope
+import numpy as np
+import pprint
+import sys
+
+
+pp = pprint.PrettyPrinter(indent=4, width=80)
+
+
+# This example is based on the C FetchForever example
```

Comment:
I don't think this is necessary. If NI-SCOPE modifies or renames or stops shipping that example, the comment will be wrong.

Reply:
Removed